src/main/java/oop/course/client/Transaction.java

```diff
@@ -0,0 +1,5 @@
+package oop.course.client;
+
+public record Transaction(String type, String from, String amount, String date) {
```

Comment:
I think it is bad since. Search for yegors Record opinion. It is anemic class (well, record..)